pkg/openstack/actuator.go

```diff
@@ -236,11 +257,35 @@ func (a *OpenStackActuator) getRootCloudCredentialsSecretData(ctx context.Contex
 	}
 
 	clouds = string(keyBytes)
-	logger.Debug("found clouds.yaml in target secret")
+	logger.Debugf("found %v in target secret", rootOpenStackCredsSecretKey)
 
 	return clouds, nil
 }
 
+func (a *OpenStackActuator) getRootCACertData(ctx context.Context, logger log.FieldLogger) (string, error) {
+	caCertConfigMap := &corev1.ConfigMap{}
+	if err := a.Client.Get(ctx, a.GetCACertConfigMapLocation(), caCertConfigMap); err != nil {
```

Comment:
Is this configmap new in 4.6? I have a 4.5 RC and do not see this configmap.
Does this code need to account for the possibility that the configmap does not exist?

Reply:
The certificate is optional, if the cluster uses self-signed one. https://docs.openshift.com/container-platform/4.5/installing/installing_openstack/installing-openstack-installer-custom.html